app/actions/capture.js

```diff
@@ -2,12 +2,15 @@ import { API }          from '../utils/api';
 import { Config }       from '../../config';
 import { checkVersion } from './utils';
 
+export const MARK_PENDING = 'MARK_PENDING';
 export const MARK_CAPTURED = 'MARK_CAPTURED';
 export const SET_CAPTURES = 'SET_CAPTURES';
 
+
```

Comment:
looks like a stray extra line here. the linter should've caught this, but it looks like CI wasn't running for forked repos, so it wasn't clear. sorry about that. i've enabled CI runs on forks now, so small things will this will be apparent immediately for you! you can also run `yarn lint` locally to see if it passes as well